packages/GA4Client/src/commerce-handler.js

```diff
@@ -63,44 +68,38 @@ CommerceHandler.prototype.logCommerceEvent = function (event) {
         return logPromotionEvent(event);
     }
 
-    ga4CommerceEventParameters = buildParameters(event);
```

Comment:
This code is duplicative.  `buildParameters` creates an object with `coupon` and `items`, but every `build` function also does that.